src/schema/v1.json

```diff
@@ -183,6 +183,7 @@
    "pkgdiff",
    "parent-pkgdiff",
    "release-payload",
+   "ibmcloud",
```

Comment:
nit: alphabetize the list?

Reply:
added a second commit to make it alphabetical